assets/js/signUp.js

```diff
@@ -0,0 +1,177 @@
+    isValid = false;
+  }
+
+  if (isValid) {
+    document.querySelectorAll('.error-message').forEach(error => {
+      error.classList.add('hidden');
+      error.classList.remove('border-red-500');
+    });
+    // Lưu accounts xuống csdl
+    const account = {
+      firstName: firstName,
+      lastName: lastName,
+      email: email,
+      checkboxStatus: document.getElementById('emailSignUp').checked,
+      password: password,
+    };
+
+    let accounts = JSON.parse(localStorage.getItem('accounts')) || [];
+    accounts.push(account);
+    localStorage.setItem('accounts', JSON.stringify(accounts));
+    showAlert(t('sign-up.success_message'), 'success');
+  }
+
+});
+
+function reloadMessageSignUp() {
+  const errorFirstName = document.getElementById('first-name-error');
+  const errorLastName = document.getElementById('last-name-error');
+  const errorEmail = document.getElementById('email-error');
+  const errorPassword = document.getElementById('password-error');
+  const errorPasswordVerify = document.getElementById('password-confirm-error');
+  const successAlert = document.getElementById('alert-message-success');
+
+  errorFirstName.textContent = t('sign-up.error_first_name');
+  errorLastName.textContent = t('sign-up.error_last_name');
+  errorEmail.textContent = t('sign-up.error_email');
+  errorPassword.textContent = t('sign-up.error_password');
+  errorPasswordVerify.textContent = t('sign-up.error_password');
+  successAlert.textContent = t('sign-up.success_message');
+}
+
+function validateInput(event) {
+  const inputElement = event.target;
+  const inputValue = inputElement.value.trim();
+  let errorElement;
+
+  if (inputElement.id === 'firstName') {
+    errorElement = document.getElementById('first-name-error');
+    if (inputValue.length >= 0) {
+      inputElement.classList.add('focus:border-primary-color');
+      inputElement.classList.remove('border-red-500');
+      errorElement.textContent = "";
+    }
+  }
+
+  if (inputElement.id === 'lastName') {
+    errorElement = document.getElementById('last-name-error');
+    if (inputValue.length >= 0) {
+      inputElement.classList.add('focus:border-primary-color');
+      inputElement.classList.remove('border-red-500');
+      errorElement.textContent = "";
+    }
+  }
+
+  if (inputElement.id === 'emailSignUp') {
+    errorElement = document.getElementById('email-error');
+    const emailPattern = /^[^\s@]+@[^\s@]+\.[^\s@]+$/;
+    if (emailPattern.test(inputValue)) {
+      inputElement.classList.add('focus:border-primary-color');
+      inputElement.classList.remove('border-red-500');
+      errorElement.textContent = "";
+    }
+  }
+
+  if (inputElement.id === 'passwordSignUp') {
+    errorElement = document.getElementById('password-error');
+    if (inputValue.length >= 0) {
+      inputElement.classList.add('focus:border-primary-color');
+      inputElement.classList.remove('border-red-500');
+      errorElement.textContent = "";
```

Comment:
```suggestion
      errorElement.textContent = '';
```
Ở code JS ưu tiên dùng nháy đơn nha Tỉnh, nếu trong text có nháy đơn thì mới dùng nháy đôi
VD: "Thông tin 'Nguyen Van A' "